prowler/providers/azure/services/search/search_client.py

```diff
@@ -0,0 +1,4 @@
+from prowler.providers.azure.services.search.search_service import Search
+from prowler.providers.common.provider import Provider
+
+search_client = Search(Provider.get_global_provider())
```

Comment:
After discussing with the team, we have concluded that it’s better to call it `AISearch`. We are aware that Microsoft refers to it simply as Search at the CLI and API levels, but we find it a bit confusing to call it just Search. Could you please update it wherever necessary?
```suggestion
search_client = AISearch(Provider.get_global_provider())
```